Mathlib/Order/SupClosed.lean

```diff
@@ -528,6 +546,41 @@ lemma InfClosed.sInf_mem_of_nonempty (hs : InfClosed s) (ht : t.Finite) (ht' : t
 
 end ConditionallyCompleteLattice
 
+section BooleanAlgebra
+variable [BooleanAlgebra α] (s : Set α)
+
+lemma compl_image_latticeClosure :
+    compl '' (latticeClosure s) = latticeClosure (compl '' s) := by
```

Comment:
Can you:
1. prove that an order iso preserves lattice closure
2. prove that `latticeClosure (ofDual ⁻¹' s) = ofDual ⁻¹' latticeClosure (`
3. deduce your `compl` result using the fact that `ofDual ∘ compl` is an order iso

Reply:
Yes there should be: You can connect both if them through `toDual` (and the proof for `toDual` should be simpler since it's basically defeq)